.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,17 @@
+---
+title: 'Question'
+labels: question
+---
+
+## Prerequisites
+
+Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**
+
+- [ ] I am running the latest version
+- [ ] I checked the documentation and found no answer
+- [ ] I checked to make sure that this issue has not already been filed
+- [ ] I'm sure that question is related to the library itself and not Bluetooth Low Energy or Classic in general. If that so, please post your question on stack overflow or one of our Gitter channels.
+
+## Question
+
+Please describe the question.
```

Comment:
Maybe add info to consider filing a stackoverflow question?

Reply:
It's mentioned above, but I will use links to be able to send question/enter channel directly.

Reply:
Ah, Ok